src/cms/templates/pages/page_tree_node.html

```diff
@@ -139,14 +139,5 @@
                 </button>
             {% endif %}
         {% endif %}
-        {% if page_translation %}
-        <a href="#" data-copy-to-clipboard="{{ page_translation.short_url }}" title="{% trans 'Copy short link' %}" class="py-3 px-2">
-            <i data-feather="copy" class="text-gray-800"></i>
-        </a>
-        {% else %}
-        <a title="{% trans 'You cannot copy a short link of a page which has no translation.' %}" class="py-3 px-2">
-            <i data-feather="copy" class="text-gray-400"></i>
-        </a>
-        {% endif %}
```

Comment:
I think I wouldn't remove this completely, but only show it when the short urls are activated and the current user is in expert mode.